documentation/1.10/content/userguide/tools/create-aux-image.md

```diff
@@ -0,0 +1,85 @@
+---
+title: "Create Auxiliary Image"
+date: 2019-02-23
+draft: false
+weight: 2
+description: "The createAuxImage command creates a new container image with WDT models, archives, and variable files."
+---
+
+
+The `createAuxImage` command helps build a container image from a given base OS image. 
+There are a number of optional parameters for the create feature. The required option for the command is marked.
+
+```
+Usage: imagetool createAuxImage [OPTIONS]
+```
+
+| Parameter | Definition | Default |
+| --- | --- | --- |
+| `--builder`, `-b` | Executable to process the Dockerfile. Use the full path of the executable if not on your path. | `docker`  |
+| `--buildNetwork` | Networking mode for the RUN instructions during the image build.  See `--network` for Docker `build`.  |   |
+| `--chown` | `userid:groupid` for JDK/Middleware installs and patches.  | `oracle:oracle` |
+| `--dryRun` | Skip Docker build execution and print the Dockerfile to stdout.  |  |
+| `--fromImage` | Container image to use as a base image when creating a new image. | `busybox`  |
+| `--httpProxyUrl` | Proxy for the HTTP protocol. Example: `http://myproxy:80` or `http:user:passwd@myproxy:8080`  |   |
+| `--httpsProxyUrl` | Proxy for the HTTPS protocol. Example: `https://myproxy:80` or `https:user:passwd@myproxy:8080`  |   |
+| `--packageManager` | Override the default package manager for the base image's operating system. Supported values: `APK`, `APTGET`, `NONE`, `YUM`, `ZYPPER`  |   |
+| `--pull` | Always attempt to pull a newer version of base images during the build.  |   |
+| `--skipCleanup` | Do not delete the build context folder, intermediate images, and failed build containers. For debugging purposes.  |   |
+| `--tag` | (Required) Tag for the final build image. Example: `store/oracle/mydomain:1`  |   |
+| `--target` | Select the target environment in which the created image will be used. Supported values: `Default` (Docker/Kubernetes), `OpenShift` | `Default`  |
+| `--wdtArchive` | A WDT archive ZIP file or comma-separated list of files.  |   |
+| `--wdtHome` | The target folder in the image for the WDT install and models.  | `/auxiliary`  |
+| `--wdtModel` | A WDT model file or a comma-separated list of files.  |   |
+| `--wdtModelHome` | The target location in the image to copy WDT model, variable, and archive files. | `{wdtHome}/models` |
+| `--wdtVariables` | A WDT variables file or comma-separated list of files.  |   |
+| `--wdtVersion` | WDT version to use.  | `latest`  |
```

Comment:
WDT version to use --> WDT version to install in `{wdtHome}/weblogic-deploy`.

Does this support a `none` option? It's conceivable MII aux image users won't need a WDT install - it could already be available in a different aux image...

Reply:
We can not assume that the WLS binary mage has WDT, most likely it will not have it.  WLS images in OCR do not have WDT.  Aux images are only for the  MII use case which require WDT.

Reply:
Updated --wdtVersion description.  No, there is no none option included at this time.

Reply:
MII use case requires WDT yes, but it can have multiple aux images, and only one of them needs to have a WDT install, if any (it could be none of them, since the main image could have the install).